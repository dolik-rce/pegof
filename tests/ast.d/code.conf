input ast.d/code.peg
ast
