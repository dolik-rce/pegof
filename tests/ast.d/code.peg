X <- "X"

# Code starts here
%%
printf("Hello world");
