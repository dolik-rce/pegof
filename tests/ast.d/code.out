GRAMMAR
  RULE X
    ALTERNATION
      SEQ
        TERM
          STRING X
  CODE  (1 comment) "printf(\"Hello world\");\n"
